Makefile

```diff
@@ -193,3 +206,8 @@ destroy_tfstate:
 destroy_tfcache:
 	find . -name .terraform -exec rm -rf {} +
 	find . -name .terraform.lock.hcl -delete
+
+.PHONY: destroy_outputs
+destroy_outputs:
+	rm -f outputs/*.sh outputs/*-kubeconfig outputs/*.jwk outputs/*.pem
```

Comment:
removing all `*.sh` should be changed to `*jumpbox.sh` and `*kubeconfig.sh` just be nice to whoever will post and create .sh scripts under the outputs folder.

Reply:
Done!